tests/expectations/tests/objc_class.rs

```diff
@@ -9,6 +9,11 @@
 extern crate objc;
 #[allow(non_camel_case_types)]
 pub type id = *mut objc::runtime::Object;
+type Class = *mut objc::runtime::Class;
+#[inline]
+fn class(name: &str) -> Class {
+    unsafe { std::mem::transmute(objc::runtime::Class::get(name)) }
```

Comment:
instead of using `transmute`, let's use something like:

```rust
objd::runtime::Class::get(name).map(|c| c as *const _ as * mut _).unwrap_or(ptr::null_mut())
```